src/faucet/constants.rs

```diff
@@ -166,6 +166,34 @@ impl FaucetInfo {
             Network::Testnet => 314159, // chainlist.org/chain/314159
         }
     }
+
+    /// Returns the maximum allowed gas limit.
+    #[allow(dead_code)]
```

Comment:
Why the `dead_code` ?.

Reply:
because it's used only on server side, hence added this to resolve the unused code warning